### DISCUSSION TITLE
<3

### DISCUSSION THREAD

**Discussion by vodkawrath** [community/community#18912] [Other Feature Feedback, Questions, & Ideas]
I'm currently studying javascript and I'm finding it a lot of fun

**Comment by melvincwng** [comment]
@vodkawrath, welcome to Github Community!

All the best in your journey to become a great javascript developer :)

**Reply by vodkawrath** [reply]
thank you!

**Comment by Andreaa-Dev** [comment]
good to hear 😊 I also like JavaScript

**Comment by Zerotask** [comment]
If you get the basics, you should hop on the TypeScript train ;)
